### PR TITLE
Script Cleanup + Few Major Changes

### PR DESCRIPTION
**Pull request by eXhumer**

### CODE REVIEW

**Comment by eXhumer** [comment]
Code has been tested with block/solid compression and decompression, all works perfectly. The comment for -o problem in main() has been removed and the problem has been fixed. Potential fix for decompress/compress tqdm not properly reaching 100% (not tested). Removed excessive whitespaces and cleaner imports for each script. Plus other possible changes I forgot to mention.

**Comment by eXhumer** [comment]
Almost all, if not all, file path related operations are now being done with pathlib instead of os.path. IMO, it is better to use pathlib for almost all, if not all, file path related operations rather than os.path.

File: nsz/NszDecompressor.py [hunk 1]
```diff
@@ -77,71 +49,57 @@ def __decompress(filePath, outputDir = None, write = True, raiseVerificationExce
...
-				
 				while i < end:
 					crypto.seek(i)
 					chunkSz = 0x10000 if end - i > 0x10000 else end - i
 					if useBlockCompression:
 						inputChunk = blockDecompressorReader.read(chunkSz)
 					else:
 						inputChunk = decompressor.read(chunkSz)
-					
+					bar.update(chunkSz/8)
```
Comment (nicoboss) [inline nsz/NszDecompressor.py]: This seems to break the progress bar when decompressing. I don't really understand why it should be progressing with chunkSz/8

Comment (eXhumer) [reply nsz/NszDecompressor.py]: Oh, sorry about that. I was just testing the tdqm update, will fix it soon on next commit to my repo.

**Comment by nicoboss** [comment]
Thanks a lot for your hard work!

**Pull request merged**